libcst/codemod/commands/tests/test_remove_unused_imports.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+
+from libcst.codemod import CodemodTest
+from libcst.codemod.commands.remove_unused_imports import RemoveUnusedImportsCommand
+
+
+class RemoveUnusedImportsCommandTest(CodemodTest):
+    TRANSFORM = RemoveUnusedImportsCommand
+
+    def test_simple_case(self) -> None:
+        before = "import a, b\na()"
+        after = "import a\na()"
+        self.assertCodemod(before, after)
+
+    def test_double_import(self) -> None:
+        before = "import a\nimport a\na()"
+        self.assertCodemod(before, before)
+
+    def test_conditional_import(self) -> None:
+        before = """
+            if True:
+                import a
+            else:
+                import b as a
+            a()
+        """
+        self.assertCodemod(before, before)
+
+    def test_unused_in_conditional(self) -> None:
+        before = """
+            if False:
+                import a
+        """
+        after = """
+            if False:
+                pass
+        """
+        self.assertCodemod(before, after)
+
+    def test_type_annotations(self) -> None:
+        before = """
+            import a
+            x: a = 1
```

Comment:
Btw, RemoveImportsVisitor uses scope analysis and it doesn't analyze string type annotation.
That means, in this example code, `import a` will be removed by this codemod.
```
import a
x: "a" = 1
```

In general, `RemoveUnusedImportsCommand` should be used with cautions since it has limitations including it doesn't consider imports used by other modules (indirect imports) and it doesn't consider references in string type annotations.
@zsol, would you like to include those as warnings as document? Probably short description in `DESCRIPTION` attr and longer explanation in docstring of `RemoveUnusedImportsCommand`.